components/CardForm/index.js

```diff
@@ -176,6 +256,7 @@ export default function CardForm({
             name="category"
             defaultValue={existingActivityData?.category || ""}
```

Comment:
If you implemented the suggestion above then you could do the following. 


```suggestion
            defaultValue={category || ""}
```